joomla/joomla-cms
# [inlinehelp] Add inlineHelp Toggle Button control to modules and plugins

### Summary of Changes

This PR adds inlineHelp toggle button to modules and plugins options to give users the capability to show/hide parameters descriptions.
This PR improves #35610 and #37849
Instead of always displaying descriptions for modules and plugins, it becomes a user choice. In case of a doubt about a parameter, just click on the button.

Note : it could be improved by adding this to all components. For example, Featured articles menu, Article Class has a 4 lines description, usefull once. 

### Testing Instructions
Apply PR #37849 before applying this PR.

- Check that inlineHelp toogle button is present and working in modules parameters.
- Check that inlineHelp toogle button is present and working in admin modules parameters.
- Check that inlineHelp toogle button is present and working in plugins parameters.
- Check that inlineHelp button is not shown if plugin has no description to display (for example content page break)

### Actual result BEFORE applying this Pull Request
After #37849 , the default behaviour is "show all descriptions" for modules and plugins parameters if no inlineHelp toggle button is displayed. 

### Expected result AFTER applying this Pull Request

InlineHelp toogle button is displayed and working for modules/plugins if parameters have descriptions. It is hidden otherwise.

### Documentation Changes Required
none
